websites/app/src/pages/Home/SubmitEntries/AddEntryModal/index.tsx

```diff
@@ -166,9 +205,18 @@ const AddEntryModal: React.FC = () => {
     [searchParams]
   )
 
+  const closeModal = () => {
+    setSearchParams((prev) => {
+      const prevParams = prev.toString()
+      const newParams = new URLSearchParams(prevParams)
+      newParams.delete('additem')
+      return newParams
+    })
+  }
+
   return (
-    <ModalOverlay>
-      <ModalContainer ref={containerRef}>
+    <ModalOverlay onClick={closeModal}>
```

Comment:
lets not close this modal if you click outside. there is sensitive information so its important the user does not accidentally loses everything by clicking outside.

in this specific modal only close it if you click on the X, like it was before